Assignment 5.Rmd

```diff
@@ -66,15 +66,18 @@ Upload the data "intelligent_tutor_new.csv". This is a data set of a differnt sa
 ```{r}
 #Upload new data
 
-D2 <- 
+D2 <- read.csv("intelligent_tutor_new.csv")
 
 #Generate predicted advice using the predict() command for new students based on tree generated from old students
 
-D2$prediction <- 
+D2$prediction <- predict(score_ctree,D2)
 
 ``` 
 ## Part III
 Compare the predicted advice with the actual advice that these students recieved. What is the difference between the observed and predicted results?
+```{r}
+mean(ifelse(D2$prediction == "no action", 1, 0))
+```
```

Comment:
Glad to see your work. It would be better if you could explain why you use mean to check the model accuracy, report the result and interpret it.